content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+1. In O3DE Editor, select the entity with the Input component in the viewport or **Entity Outliner**.
+
+1. In Entity Inspector, click **Add Component** and add the **Script Canvas** component.
+
+1. In the Script Canvas component, for **Script Canvas Asset**, click {{< icon "open-in-internal-app.svg" >}}, and then create a new Script Canvas graph like the following example.
+
+### Example Script Canvas graph
+
+In the following graph, the **InputHandler** node connects `Action` events to various **Transform** nodes. The Input component sends **Pressed**, **Held**, and **Released** events to the **InputHandler** node When the state of an event generator changes.
+
+![Example Script Canvas graph for the Input component](/images/user-guide/interactivity/input/sc-input-example.png)
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to input events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In O3DE Editor, select the entity with the Input component in the viewport or **Entity Outliner**.
+
+1. In Entity Inspector, click **Add Component**, and then add the **Lua Script** component.
+
+1. In the Lua Script component, click {{< icon "open-in-internal-app.svg" >}}.
+
+1. In Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an input event that you want to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the input event. The second parameter is an `InputEventNotificationId` for the **Event Name**.
+
+### Example Lua script
+
+The following Lua script connects `Action` events to the individual functions `OnPressed`, `OnHeld`, and `OnReleased`.
+
+```lua
+local tutorial_input = {
+    Properties = {}
+}
+function tutorial_input:OnActivate()
+    self.ActionInputs = {}
+
+    self.ActionInputs.OnPressed = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 2.0)
+    end
+
+    self.ActionInputs.OnHeld = function(_, value)
+        TransformBus.Event.RotateAroundLocalZ(self.entityId, 0.01)
+    end
+
+    self.ActionInputs.OnReleased = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 1.0)
+    end
+
+    self.ActionInputHandler = InputEventNotificationBus.Connect(self.ActionInputs, InputEventNotificationId("Action"))
+end
+
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect the input event in your `.inputbindings` file to an input handler, you can test player input in **O3DE Editor**.
```

Comment:
```suggestion
After you use either Script Canvas or Lua to connect the input event in your `.inputbindings` file to an input handler, you can test player input in O3DE Editor.
```
This occurrence of "O3DE Editor" is well after the first usage, so it doesn't need bold formatting.